CONTRIBUTING.md

```diff
@@ -646,192 +642,200 @@ If you have any problems with formatting, please ping the [formatting team](http
+To contribute effectively and efficiently, you need to be aware of how the process generally works.
+This section aims to document the process as we live it in Nixpkgs to set the right expectations and give practical tips on how to work with it.
 
 ## I opened a PR, how do I get it merged?
 [i-opened-a-pr-how-do-i-get-it-merged]:#i-opened-a-pr-how-do-i-get-it-merged
 
-In order for your PR to be merged, someone with merge permissions on the repository ("committer") needs to review and merge it.
-Because the group of people with merge permissions is mostly a collection of independent unpaid volunteers who do this in their own free time, this can take some time to happen.
+In order for your PR to be merged, a committer needs to review and merge it.
+Because committers are mostly independent, unpaid volunteers, this can take time.
 It is entirely normal for your PR to sit around without any feedback for days, weeks or sometimes even months.
-We strive to avoid the latter cases of course but the reality of it is that this does happen quite frequently.
-Even when you get feedback, follow-up feedback may take similarly long.
-Don't be intimidated by this and kindly ask for feedback again every so often.
-If your change is good it will eventually be merged at some point.
+We strive to avoid this, but the reality is that it happens frequently.
+Even when you get feedback, follow-ups may take just as long.
+Don't be intimidated and kindly ask for feedback again every so often.
+If your change is good, it will eventually be merged.
 
-There are some things you can do to help speed up the process of your PR being merged though.
-In order to speed the process up, you need to know what needs to happen before a committer will actually hit the merge button.
-This section intends to give a little overview and insight of what happens after you create your PR.
+You can often speed up the process by understanding the committer's perspective and preparing your PR with reviewing in mind.
 
 ### The committer's perspective
 
 PRs have varying quality and even the best people make mistakes.
-It is the role of the committer team to assess whether any PR's changes are good changes or not.
-In order for any PR to be merged, at least one committer needs to be convinced of its quality enough to merge it.
+Committers need to assess whether a PR's changes are good or not.
+To merge, at least one committer has to be convinced of its quality enough.
 
-Committers typically assess three aspects of your PR:
+Committers typically assess three aspects:
 
-1. Whether the change's intention is necessary and desirable
-2. Whether the code quality of your changes is good
-3. Whether the artefacts produced by the code are good
+1. Whether the change's intention is necessary and desirable.
+2. Whether the code quality of your changes is good.
+3. Whether the produced artifacts are good.
```

Comment:
This last bullet point feels _very_ vague. I'm not convinced "artifacts" is a useful term here to communicate what it's trying to say.

I'm not even sure whether it's saying "packages built" or "the final code, after applying the PR's patches", or something else...

Not really an issue with your change, just that the existing wording is poor.

I believe the three bullet points are trying to say:

1. intention/motivation/justification necessity/desirability
2. patches/diff quality
3. final/merged quality

But I'm not 100% sure.

Reply:
> For a package, the artifact would be the result of the build. Does the build even work? Do the produced binaries work? etc.
> 
> For a NixOS module, this is different.
> 
> For a new function in `lib` - entirely different, again.
> 
> I think this is what is meant with "artifacts". Aka the "result of what the code _does_".

That makes sense. But I guess my confusion highlights that the terminology may not be clear to new contributors.

Not really an issue with your PR, but with the existing wording.